go/billing/tasks.py

```diff
@@ -547,7 +548,9 @@ def create_low_credit_notification(
             'reference': notification.id,
         })
 
-    email = EmailMessage(subject, message, email_from, [email_to])
+    # remove duplicate emails from the recipient list
+    recipient_emails = list(set([email_to, support_email]))
+    email = EmailMessage(subject, message, email_from, recipient_emails)
```

Comment:
As discussed on slack, we probably want the `support_email` to be a `bcc`. Should be as simple as adding `bcc=support_email` to the list of args, and fixing the relevant test.